src/pages/api/auth/[...nextauth].ts

```diff
@@ -1,4 +1,20 @@
 import NextAuth from "next-auth";
-import { authOptions } from "~/server/auth";
+import GoogleProvider from "next-auth/providers/google";
+import { PrismaAdapter } from "@next-auth/prisma-adapter";
+import { PrismaClient } from "@prisma/client";
 
-export default NextAuth(authOptions);
+const prisma = new PrismaClient();
```

Comment:
what's this? why not use the authOptions object and edit it there if you need to